src/cargo/core/resolver/mod.rs

```diff
@@ -244,149 +190,240 @@ struct Context {
 /// Builds the list of all packages required to build the first argument.
 pub fn resolve(summary: &Summary, method: &Method,
                registry: &mut Registry) -> CargoResult<Resolve> {
-    trace!("resolve; summary={}", summary.package_id());
+    debug!("resolve; summary={}", summary.package_id());
```

Comment:
Ah you can actually leave these as `trace!` because that's defined in liblog as a separate logging level, the special part for this module was the guard with `cfg!(debug_assertions)`